services/source-debugger/eventUploader.go

```diff
@@ -148,7 +148,9 @@ func uploadEvents(eventBuffer []*GatewayEventBatchT) {
 			sentAt := getTimestampFromEvent(ev, "sentAt")
 
 			ev["originalTimestamp"] = originalTimestamp.Format(time.RFC3339)
-			ev["timestamp"] = misc.GetChronologicalTimeStamp(receivedAtTS, sentAt, originalTimestamp).Format(time.RFC3339)
+			if _, ok := ev["timestamp"]; !ok {
```

Comment:
Source debugger shows the events received at the gateway. The `timestamp` is computed later in the processor.
I don't think we should even send our computed timestamp here. 

In fact, we should not add any new fields here in eventUploader. Just send the event as it is received.
We can remove lines from 143 to 153. Let's just append ev to array.

Check with @prabrisha-rudder once, since she made the changes last time. I might have missed something.